.travis.yml

```diff
@@ -104,7 +105,10 @@ script:
   - make -j2
   - make -j2 prove-all PROVE_OPTS=-v
   - if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_CPU_ARCH" == "amd64" ]]; then
-        clang-tidy --version && make -j2 tidy;
+        clang-tidy --version && TIDY_OPTS="-fix-errors" make -j2 tidy && git diff --exit-code;
```

Comment:
A few times we've have `clang-tidy` fail but `make tidy` not forward the error code. This should catch that.